db/migrate/20191025141802_create_application_experience.rb

```diff
@@ -0,0 +1,22 @@
+class CreateApplicationExperience < ActiveRecord::Migration[6.0]
+  def change
+    create_table :application_experiences do |t|
+      t.references :application_form, null: false, foreign_key: { on_delete: :cascade }
+
+      t.string :type, null: false
+      t.string :title, null: false
+      t.string :organisation, null: false
+
+      t.text :details, null: false
+
+      t.boolean :working_with_children, null: false
+
+      t.datetime :start_date, null: false
+      t.datetime :end_date, null: false
+
+      t.string :commitment
```

Comment:
I think commitment can be rolled into details.
On the prototype UI the time-commitments is a text field that is also used to give role details and responsibilities.

Reply:
Discussed offline, we thought through it and in the end it's not bad to leave it as a separate field which is null in the case of volunteering experience.